packages/terra-search-field/tests/wdio/search-field-spec.js

```diff
@@ -68,12 +68,13 @@ Terra.describeViewports('Search Field', ['medium'], () => {
 
     Terra.it.validatesElement('default value');
 
-    it('should enter a new search term', () => {
-      browser.click('input');
-      for (let i = 0; i < 7; i += 1) {
-        browser.keys('Backspace');
-      }
+    it('should clear the value', () => {
+      browser.click('#searchfield [data-terra-search-field-button="Clear"]');
+    });
+
+    Terra.it.validatesElement('Clear');
```

Comment:
Terra.it helpers are getting deprecated, use `Terra.validates.element()` inside the it block.

Reply:
updated here https://github.com/cerner/terra-core/pull/3207/commits/3719667d69adc3eef032db7b37867b2ee9c928ed